.dependabot/config.yml

```diff
@@ -4,6 +4,31 @@ update_configs:
   - package_manager: 'javascript'
     directory: '/'
     update_schedule: 'weekly'
```

Comment:
> The downside of this is that if we mark a bunch of dependencies as "squash when passing" all at the same time, repo ranger really crunches through build minutes (this is a problem for both heroku and netlify). Doing this is might make that problem worse rather than better, particularly if we get a week with a lot of matching updates 😬

Is it possible to update some (automerged) dependencies daily and other dependencies weekly? If not we can update all dependencies daily.

Reply:
I think everything has to be on the same schedule.

I think moving to daily updates is going to give us a higher volume of dependency PRs requiring manual review for stuff like the gatsby* and babel* dependencies which are cranking out multiple releases per week.

Reply:
Another thing that will (eventually) be a big help is the ability to have dependency updates grouped together within a single PR - https://github.com/dependabot/feedback/issues/5